libbeat/scripts/Makefile

```diff
@@ -377,6 +377,7 @@ build-image: write-environment
 # To use it for running the test, set ES_HOST and REDIS_HOST environment variable to the ip of your docker-machine.
 .PHONY: start-environment
 start-environment: stop-environment
+	${DOCKER_COMPOSE} pull --include-deps
```

Comment:
I assume this will use the cached layers if they are available and identical?

Reply:
Yes, it still reuses local layers.

Reply:
But I see build times increased some minutes... maybe this is normal if we were re-using images too much.

Reply:
I think its ok as it gets us on the same page with travis. also if intermediary images used, the increase should only be once a day